src/renderer/components/+helm-charts/icon.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import React, { useState } from "react";
+import type { HelmChart } from "../../../common/k8s-api/endpoints/helm-charts.api";
+
+export interface HelmChartIconProps {
+  className?: string;
+  chart: HelmChart;
+}
+
+export const HelmChartIcon = ({
+  chart,
+  className,
+}: HelmChartIconProps) => {
+  const [failedToLoad, setFailedToLoad] = useState(false);
+  const icon = chart.getIcon();
+
+  if (!icon || failedToLoad) {
+    return (
+      <div className={className}>
+        <svg viewBox="0 0 722.8 702" xmlns="http://www.w3.org/2000/svg">
```

Comment:
nit: This `svg` should be in separate file. Now it's impossible to preview.

Reply:
Yeah... @aleksfront converted move this instead of the previous version which `dangerouslySetInnerHTML`. I think having a component plus svg file would be best.